wheel/src/api.rs

```diff
@@ -1,25 +1,51 @@
+use super::lazy_node::LazyNode;
+use crate::adapt_response::adapt_response;
+use clvmr::allocator::Allocator;
+use clvmr::chia_dialect::ChiaDialect;
+use clvmr::cost::Cost;
+use clvmr::reduction::Response;
+use clvmr::run_program::run_program;
+use clvmr::serde::{node_from_bytes, serialized_length_from_bytes};
+use clvmr::{LIMIT_HEAP, LIMIT_STACK, MEMPOOL_MODE, NO_NEG_DIV, NO_UNKNOWN_OPS};
 use pyo3::prelude::*;
 use pyo3::wrap_pyfunction;
 
-use super::lazy_node::LazyNode;
-use super::run_program::{
-    __pyo3_get_function_deserialize_and_run_program2, __pyo3_get_function_run_chia_program,
-    __pyo3_get_function_serialized_length,
-};
-use clvmr::{LIMIT_HEAP, NO_NEG_DIV, NO_UNKNOWN_OPS, LIMIT_STACK, MEMPOOL_MODE};
+#[pyfunction]
+pub fn serialized_length(program: &[u8]) -> PyResult<u64> {
+    Ok(serialized_length_from_bytes(program)?)
+}
+
+#[pyfunction]
+pub fn run_serialized_program(
+    py: Python,
+    program: &[u8],
+    args: &[u8],
+    max_cost: Cost,
+    flags: u32,
+) -> PyResult<(PyObject, LazyNode)> {
```

Comment:
it would be nice to have CI actually exercise the success and failure return paths from this function too